spylib/token.py

```diff
@@ -400,3 +401,11 @@ def load(cls, store_name: str):
         therefore the developer should override this.
         """
         pass
+
+
+def verify_webhook(data: str, hmac_header: str, api_secret_key: str) -> bool:
```

Comment:
If you return a boolean, I think a better name would be this:
```suggestion
def is_webhook_valid(data: str, hmac_header: str, api_secret_key: str) -> bool:
```

This way one can have a readable if statement:
```python
if is_webhook_valid():
```